hamilton/parameter_input/lhs_sampling.py

```diff
@@ -49,9 +49,19 @@
     default="lhs_params.csv",
     help="Output file path, relative to current working directory",
 )
-def cli(num_param_sets, output_file):
+@click.option(
+    "--max-options-per-param",
+    "-m",
+    default=None,
+    type=click.IntRange(
+        1,
+    ),
+    help="Generate no more than m different values per param.",
```

Comment:
May be worth adding a note in the help string/documentation saying this means we don't get true maximin LHS?